### DISCUSSION TITLE
Tieba Remix 0.4.2 BETA

### DISCUSSION THREAD

**Discussion by 0xb1aded** [WiresawBlade/Tieba-Remix#23] [Announcements]
### 📢 通知

- 项目将不会对贴吧原版广告做任何处理，也不会对这些内容进行非必要样式修改。如有需要，请使用广告拦截插件
- 不知不觉又要过年了，本项目的开工时间就是去年的年三十，即将迎来第一个周年。在此祝大家新年快乐

### ⚠ 注意

- 该项目处于早期开发阶段，仍有大量工作未完成，无法保证当前的使用体验，请谨慎安装
- 这是一个 BETA 版本，功能有待完善且可能会产生未知错误，该版本的特性也不保证一定会保留到后续版本中

### ⚡ 功能

- 支持设置自定义字体，包括 主要字体组合 与 等宽字体组合。在 `显示 -> 字体` 查看详情

### 🚀 其他

- 优化文件大小
- 提升了首屏加载性能，尽量降低闪屏概率
- 提升了新版看贴页面的性能
- 基于 ElementPlus 的组件以切换至中文
- 脚本现在只会对贴吧中已经手动适配的页面生效
- 部分样式调整
- 优化了 自动主题 和 自定义背景 的实现
- 现在手动主题的滚动条的基本组件都会跟随主题变化，而不是只能跟随系统设置 (本身是 Edge 的 bug，由于近期该问题已修复，所以不再屏蔽此特性)

### 🐛 修复

- 移除帖子页面和 `图片查看器` 组件的绑定
- 针对页面宽度变化而动态布局的 `收缩视图` 功能已恢复
- 修复设置组件中 关于页面 开源协议显示错误的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/WiresawBlade/Tieba-Remix/releases/tag/v0.4.2-beta'>Tieba Remix 0.4.2 BETA</a>.</em>

**Comment by noodles500kg** [comment]
EDGE好像无法加载最新版本，chrome都正常，好奇怪。
